projection/core/src/main/scala/com/lightbend/lagom/projection/State.scala

```diff
@@ -57,6 +94,16 @@ object Projection {
   def apply(name: String, workers: Seq[Worker]) = new Projection(name, workers)
 }
 
+/**
```

Comment:
It looks like this section:

> Each worker includes information about the particular tagname it is trackign. Note
> that multiple workers may track the same tagName because each worker is part of a different projection.
> Each worker also has a key which is unique across the whole cluster. Finally, the data related to a
> worker that's part of the State includes a requested status and an observed status for the worker.

Is more about `Worker` than the `State`?

Reply:
Hmmm, yeah...
Since `State` is the aggregate root I wanted to give an overview of the information it holds as a summary. Then, in the docs for `Worker`I got into more detail.